src/models/FGPropagate.cpp

```diff
@@ -882,6 +882,11 @@ void FGPropagate::bind(void)
   PropertyManager->Tie("attitude/pitch-rad", this, (int)eTht, (PMF)&FGPropagate::GetEuler);
   PropertyManager->Tie("attitude/heading-true-rad", this, (int)ePsi, (PMF)&FGPropagate::GetEuler);
 
+  PropertyManager->Tie("attitude/quaternion-0", this, 1, (PMF)&FGPropagate::GetQuaternion);
+  PropertyManager->Tie("attitude/quaternion-1", this, 2, (PMF)&FGPropagate::GetQuaternion);
```

Comment:
Can you change the name to local-quaternion-0

Reply:
done